library/almalinux

```diff
@@ -18,7 +18,7 @@ arm64v8-File: Dockerfile-aarch64-minimal
 ppc64le-File: Dockerfile-ppc64le-minimal
 Architectures: amd64, arm64v8, ppc64le
 
-Tags: latest, 8, 8.6, 8.6-20221101
+Tags: 8.6, 8.6-20221101
```

Comment:
Any particular reason to keep this version (and `8.6-minimal`) listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Our general rule-of-thumb is that if they aren't going to be updated in the future, they should be removed (ie, if there won't ever be a new `8.6-2022MMDD`, and it will only be `8.7` going forward).

Reply:
Done. No more `8.6` releases